### PR TITLE
[VITE] Moved react/react-dom to peerDependencies

### PR DESCRIPTION
**Pull request by rubenthoms**

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -189,5 +189,9 @@
     "engines": {
         "node": ">=14.17",
         "npm": ">=6.14"
+    },
```
Comment (hkfb) [inline react/package.json:192]: This is already done. I think we should aim for `"react": ^17 || ^18` but we should wait until we have completed migration to MUI 5.

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/package.json (hunk 1 above)
Comment (rubenthoms) [reply react/package.json:192]: @hkfb Can we close this PR then?

Comment (hkfb) [reply react/package.json:192]: Yes.

Comment (t0oF-azpn) [reply react/package.json:192]: Yes.

**Comment by hkfb** [comment]
Closing this since it's already fixed.

**Pull request closed without merging**